shared/chat/conversation/attachment-get-titles/index.tsx

```diff
@@ -96,89 +83,174 @@ class GetTitles extends React.Component<Props, State> {
     if (!info) return null
 
     return (
-      <Kb.ScrollView style={{height: '100%', width: '100%'}}>
-        <Kb.Box style={Styles.isMobile ? stylesMobile : stylesDesktop}>
-          <Kb.Box style={{...Styles.globalStyles.flexBoxCenter, height: 150, width: 150}}>
-            {info.type === 'image' ? (
-              <Kb.OrientedImage
-                src={Styles.isAndroid ? `file://${path}` : path}
-                style={Styles.isMobile ? {height: 150, width: 150} : {maxHeight: '100%', maxWidth: '100%'}}
+      <Kb.ScrollView style={styles.scrollView}>
```

Comment:
:sheep: on iOS this extends down behind the keyboard, so you can't get to the button without blurring the input (on the 5S sim). Would be nice for this to stay in the `KeyboardAvoidingView`
<img src="https://user-images.githubusercontent.com/11968340/59461975-a3b92c80-8df0-11e9-9399-7d6b89cc60a9.gif" width="250" />

Reply:
Interesting. I was testing on the 5s emulator and never ran into this. I'll take a look.